conduit/plugins/processors/filterprocessor/gen/generate.go

```diff
@@ -60,6 +60,8 @@ var ignoreTags = map[string]bool{
 	"txn.apsu": true,
 	"dt.lg":    true,
 	"dt.sa":    true,
+	// Filter out heartbeat fields that are not obvious how to filter upon
+	"txn.hb": true,
```

Comment:
I have no idea what's going on here, so I'm guessing here. But is seems like the hb address is a worthwhile and understandable field. I guess the rest of the hb fields are not very interesting.